### PR TITLE
IA-1625: allow run multiple workflow at the same time

### PR DESCRIPTION
**Pull request by codeKameleon**
Explain what problem this PR is resolving

## Self proof reading checklist

- [x] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] Did I add translations
- [ ] My migrations file are included
- [ ] Are there enough tests
## Changes

Explain the changes that were made. The idea is not to list exhaustively all the changes made (Github already provides a full diff), but to help the reviewers better understand:
- which specific file changes go together, e.g: when creating a table in the front-end, there usually is a config file that goes with it
- the reasoning behind some changes, e.g: deleted files because they are now redundant
- the behaviour to expect, e.g: tooltip has purple background color because the client likes it so, changed a key in the API response to be consistent with other endpoints

## How to test

Explain how to test your PR.
If a specific config is required explain it here (dataset, account, profile, ...)

## Print screen / video

Upload here print screens or videos showing the changes

## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
 
 concurrency: 
   group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
+  cancel-in-progress: false
```
Comment (quang-le) [inline .github/workflows/cypress.yml:9]: I think this reverts a change @qgerome  made to give us some performance gain

**Review by quang-le — COMMENTED** [review]
Pls check your change with @qgerome

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (qgerome) [reply .github/workflows/cypress.yml:9]: This change in particular was not for performance but to cancel already running workflows when you trigger a new one. But there is an error in the `group` and it cancels all runs of this workflow instead. We can safely merge this.

**Review by qgerome — APPROVED** [review]

**Comment by codeKameleon** [comment]
@cypress

**Comment by codeKameleon** [comment]
@cypress

**Pull request closed without merging**